dev/sparktestsupport/modules.py

```diff
@@ -35,10 +73,9 @@ class Module(object):
     def __init__(self, name, dependencies, source_file_regexes, build_profile_flags=(),
                  environ=None, sbt_test_goals=(), python_test_goals=(),
                  excluded_python_implementations=(), test_tags=(), should_run_r_tests=False,
-                 should_run_build_tests=False):
+                 should_run_build_tests=False, python_test_paths=(), python_excluded_tests=()):
         """
         Define a new module.
-
```

Comment:
I think we should keep this newline between description and parameter descriptions.